fastlane/lib/fastlane/erb_template_helper.rb

```diff
@@ -24,7 +24,13 @@ def initialize(hash, trim_mode = nil)
     end
 
     def render(template)
-      ERB.new(template, nil, @trim_mode).result(binding)
+      # From Ruby 2.6, ERB.new takes keyword arguments and positional ones are deprecated
+      # https://bugs.ruby-lang.org/issues/14256
+      if RUBY_VERSION >= "2.6.0"
```

Comment:
This might not be the best way to compare Ruby versions? 

```
# If RUBY_VERSION was 2.5.4, this would evaluate to false
RUBY_VERSION < "2.10.0"
```

☝️  This is not going to happen (I think) but it might be more correct to....

```suggestion
      if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("2.6.0")
```

Reply:
That's true, haha🙂